solutions/bronze/usaco-1251.mdx

```diff
@@ -0,0 +1,84 @@
+
+By sorting the array, we can have the arrangement beginning with the smallest fee that the cow wills pay and ending with the biggest fee that the cow wills pay.
+
+We can easily calculate the amount of total money that can be generated by this array because the index of the array and the ```max tuition``` is related.
+
+For example, starting with the zero position of the array, by multiplying the number of cows that will pay a bigger amount of tuition compared to the cow at the zero position with the tuition that the cow at the zero position wills pay, we can easily know the ```best tuition``` that the cows can pay.
+
+After calculating the zero position, by moving to the next position which is the first position, again we can simply multiply the number of cows that will pay a bigger amount of tuition compared to the cow at the first position, which will be substracting -1 from the value of the previous example, with the tuition that the cow at the first position wills to pay.
+
+By comparing the first and second values that we have calculated through this example, we can know the ```best tuition``` that the cows can pay.
+
+The number of cows that will pay bigger tuition than the cow at a certain index could be easily calculated through sorting, and this is the point why sorting is crucial in this problem.
+
+By sorting it, we can know the value of the number of cows that will pay bigger tuition that the cow at a certain index by subtracting the index value from the length of the array.
+
+For example, if I want to find out how many cows are willing to pay the tuition that the cow at index 2 wants to pay, I could easily do this by sorting the array and subtracting 2 from the length of the array because all of the cows that have a bigger index than 2 will be able to pay the fee since we sorted it.
+
+By comparing these values, we can have the maximum amount of tuition that the cows can pay which is indicated as the ```best tuition```.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(Nlog N)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int main() {
+	int n;
+	cin >> n;
+	long long maxTuition[n];
+	for (int i = 0; i < n; i++) { cin >> maxTuition[i]; }
+
+	sort(maxTuition, maxTuition + n);
+
+	long long maxMoney = 0,
+	          bestTuition =
+	              0;  // Need to use long long form rather than int as the 10^5
+	                  // times 10^6 exceeds the range of integer.
+	for (int i = 0; i < n; i++) {
+		long long setTuitionToIthCow =
+		    maxTuition[i] *
+		    (n - i);  // (n - i) is used to calculate the number of cows that
+		              // are willing to pay the fee.
+		if (setTuitionToIthCow >
+		    maxMoney) {  // the if statement changes the best tuition only if
+			             // the calculated value is bigger than the maximum
+			             // value that is calculated throughout the for loop.
+			maxMoney = setTuitionToIthCow;
```

Comment:
the formatting of the comments makes the code harder to read, could you reformat this & make it a multiline comment (if needed)?

Reply:
Changed it